Compile with socket.io-client

I try compile a simple project with one file test.js and one simple line
--------

```javascript
  console.log('Hello world');
```

```
nexe -i test.js -o test.exe  
```

Compile - OK. I get correct file test.exe


I add socket.io-client and try compile again:
--------

```javascript
var client = require('socket.io-client');  
  console.log('Hello world');
```

```
nexe -i test.js -o test.exe   
```

But error occuried:
--------

```
----> bundle Z:\Temp\!\Socket.io\test.js  
----> bundle -> Z:\Temp\!\Socket.io\tmp\nexe\latest\node-v0.10.30\lib\nexe.js  
----> make  
Found ctrpp in WinSDK--will build generated files into tools/msvs/genfiles.  
{ 'target_defaults': { 'cflags': [],  
                       'default_configuration': 'Release',  
                       'defines': [],  
                       'include_dirs': [],  
                       'libraries': []},  
  'variables': { 'clang': 0,  
                 'host_arch': 'x64',  
                 'node_has_winsdk': 'true',  
                 'node_install_npm': 'true',  
                 'node_prefix': '',  
                 'node_shared_cares': 'false',  
                 'node_shared_http_parser': 'false',  
                 'node_shared_libuv': 'false',  
                 'node_shared_openssl': 'false',  
                 'node_shared_v8': 'false',  
                 'node_shared_zlib': 'false',  
                 'node_tag': '',  
                 'node_unsafe_optimizations': 0,  
                 'node_use_dtrace': 'false',  
                 'node_use_etw': 'true',  
                 'node_use_openssl': 'true',  
                 'node_use_perfctr': 'true',  
                 'node_use_systemtap': 'false',  
                 'python': 'C:\\Python27\\python.exe',  
                 'target_arch': 'ia32',  
                 'v8_enable_gdbjit': 0,  
                 'v8_no_strict_aliasing': 1,  
                 'v8_use_snapshot': 'true',  
                 'visibility': '',  
                 'want_separate_host_toolset': 1}}  
creating  config.gypi  
creating  config.mk  
Project files generated.  
  http_parser.vcxproj -> Z:\Temp\!\Socket.io\tmp\nexe\latest\node-v0.10.30\Release\lib\http_parser.lib  
  cares.vcxproj -> Z:\Temp\!\Socket.io\tmp\nexe\latest\node-v0.10.30\Release\lib\cares.lib  
  libuv.vcxproj -> Z:\Temp\!\Socket.io\tmp\nexe\latest\node-v0.10.30\Release\lib\libuv.lib  
  node_js2c  
  non-ascii value lib\nexe.js:3339:27  
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\Microsoft.CppCommon.targets(151,5): error MSB6006: "cmd.exe" returned with code 1. [Z:\Temp\!\Socket.io\tmp\nexe\latest\node-v0.10.30\node_js2c.vcxproj]  
  v8_base.vcxproj -> ..\..\..\..\build\Release\lib\v8_base.lib  
  zlib.vcxproj -> Z:\Temp\!\Socket.io\tmp\nexe\latest\node-v0.10.30\Release\lib\zlib.lib  
  openssl.vcxproj -> Z:\Temp\!\Socket.io\tmp\nexe\latest\node-v0.10.30\Release\lib\openssl.lib  
----> cp Z:\Temp\!\Socket.io\tmp\nexe\latest\node-v0.10.30\Release\node.exe Z:\Temp\!\Socket.io\test.exe  
undefined  
```

--- a/lib/bundle.js
+++ b/lib/bundle.js
@@ -62,7 +62,8 @@ function bundle (input, complete) {
         filter: function (id) {
           if (id == 'nexeres') { return false; } //treat our new moudle as a builtin module
           return !~builtins.indexOf(id);
-        }
+        },
+        extensions: ['.js', '.json' ]
       });
       md.pipe(through(function (chunk) {
         deps.push(chunk);

